frontend/tests/unit/fragments/public-schools/school-level-totals-test.js

```diff
@@ -0,0 +1,195 @@
+
+    assert.equal(school_level_totals.subdistricts.capacityNoActionTotal, 400);
+
+  });
+
+  test('#capacityWithActionTotal is calculated correctly', function(assert) {
+    let school_level_totals = run(() =>
+    SchoolLevelTotals.create({
+      capacityNoActionTotal: 100,
+      newSchoolSeats: 300
+    }));
+
+    assert.equal(school_level_totals.capacityWithActionTotal, 400);
+
+  });
+
+  test('#seatsNoActionTotal is calculated correctly', function(assert) {
+    let school_level_totals = run(() =>
+    SchoolLevelTotals.create({
+      capacityNoActionTotal: 400,
+      enrollNoActionTotal: 300
+    }));
+
+    assert.equal(school_level_totals.seatsNoActionTotal, 100);
+
+  });
+
+  test('#seatsWithActionTotal is calculated correctly', function(assert) {
+    let school_level_totals = run(() =>
+    SchoolLevelTotals.create({
+      capacityWithActionTotal: 400,
+      enrollWithActionTotal: 300
+    }));
+
+    assert.equal(school_level_totals.seatsWithActionTotal, 100);
+
+  });
+
+  test('#utilizationNoActionTotal is calculated correctly', function(assert) {
+    let school_level_totals = run(() =>
+    SchoolLevelTotals.create({
+      enrollNoActionTotal: 177,
+      capacityNoActionTotal: 145
+    }));
+
+    assert.equal(school_level_totals.utilizationNoActionTotal, 1.2207);
+
+  });
+
+  test('#utilizationWithActionTotal is calculated correctly', function(assert) {
+    let school_level_totals = run(() =>
+    SchoolLevelTotals.create({
+      enrollWithActionTotal: 177,
+      capacityWithActionTotal: 145
+    }));
+
+    assert.equal(school_level_totals.utilizationWithActionTotal, 1.2207);
+
+  });
+
+  test('#utilizationChangeTotal is calculated correctly', function(assert) {
+    let school_level_totals = run(() =>
+    SchoolLevelTotals.create({
+      utilizationWithActionTotal: 100.66666,
+      utilizationNoActionTotal: 50.55555
+    }));
+
+    assert.equal(school_level_totals.utilizationChangeTotal, 50.1111);
+
+  });
+
+  test('#impact is calculated correctly', function(assert) {
+    let school_level_totals = run(() =>
+    SchoolLevelTotals.create({
+      utilizationChangeTotal: 1,
+      utilizationWithActionTotal: 0.8
+    }));
+
+    assert.equal(school_level_totals.impact, false);
+
+  });
+
+  test('#mitigateSeatCount is calculated correctly', function(assert) {
+    let school_level_totals = run(() =>
+    SchoolLevelTotals.create({
+      enrollWithActionTotal: 7,
+      capacityWithActionTotal: 2,
+      utilizationNoActionTotal: 3
+    }));
+
+    // seatsToMitigateUtilization = enrollWithActionTotal - capacityWithActionTotal - 1
+    // seatsToMitigateUtilization = 7 - 2 - 1 = 4
+    // seatsToMitigateChange = (enrollWithActionTotal / (utilizationNoActionTotal + 0.0499)) - capacityWithActionTotal
+    // seatsToMitigateChange = (7 / (3 + 0.0499)) - 2 = 0.295157
+    // Math.ceil(seatsToMitigateChange) = 1
+    // seatsToMitigateUtilization > seatsToMitigateChange, SO RETURN seatsToMitigateChange,
```

Comment:
Should this be deleted? Not sure it adds clarity

Reply:
I added this for the tester's purpose. It helped me work through what the end values should be and I feel like it'll help future testers. Also could just save a little time for folks looking at the code in the future. I'd love to keep it there

Reply:
Sure, whatever helps - I just thought it was commented-out code.

Reply:
ah gotcha!